generic/proof-shell.el

```diff
@@ -1712,7 +1716,7 @@ i.e., 'goals or 'response."
 	    (buffer-substring-no-properties rstart gmark)))
        ;; display goals output second so it persists in 2-pane mode
        (unless (memq 'no-goals-display flags)
-	 (pg-goals-display proof-shell-last-goals-output both))
+	 (pg-goals-display proof-shell-last-goals-output t))
```

Comment:
I quickly tested locally and it seems it works with `both` instead of `t`. Can you please revert this and re-test?